frontend/apps/case-exploration/src/components/table-filters/case-exploration-table-filters.tsx

```diff
@@ -35,6 +35,28 @@ const CRITERIAS = {
   request_id: { key: 'request_id', weight: 9 },
 };
 
+/**
+ * Case autocomplte
+ */
+async function fetchAutocompleteCases(prefix: string, minSearchLength: number) {
+  if (!prefix || prefix.length < minSearchLength) {
+    return {};
+  }
+  const response = await caseApi.autocompleteCases(prefix, '10');
+  if (response?.data && response.data.length > 0) {
+    const grouped = response.data.reduce((acc, result) => {
+      if (!acc[result.type]) {
+        acc[result.type] = [];
+      }
+      acc[result.type].push(result);
+      return acc;
+    }, {} as GroupedAutocompleteResults);
+    return grouped; // Add explicit return
+  }
+
+  return {}; //
+}
```

Comment:
why not reverse the condition here and only return {} one time?

Reply:
Je pense que l'ordre est bon, la première condition évite un appel. la seconde est si l'appel ne retourne rien